libs/documentation/src/lib/components/workspace-tier-2/item/item.module.ts

```diff
@@ -12,7 +12,6 @@ import { ComponentWrapperComponent } from './../../../shared/component-wrapper/c
 import { ResultsModule } from './demos/results/results.module';
 import { ResultsComponent } from './demos/results/results.component';
 
-
 declare var require: any;
```

Comment:
This file seems like it was created for layouts demo - can we delete this or is there value in keeping it?